components/Logo.vue

```diff
@@ -1,5 +1,12 @@
 <template>
-  <span class="dark:text-highlight text-sky-700">LU</span>
-  <span class="dark:text-highlight2 text-sky-600">Comp</span>
-  <span class="dark:text-highlight3 text-sky-500">Soc</span>
+  <div style="display:flex;">
+    <img src="/img/logo/light.svg" alt="CompSoc Logo"/>
```

Comment:
Given that the image isn't really adding anything with the title after it, might it be better leaving the alt blank (like for decorative elements), so screen readers etc see "LUCompSoc" instead of "CompSoc Logo LUCompSoc"?

Reply:
The problem with this is that Google hates images without alt text